MastodonSDK/Sources/MastodonUI/View/Content/StatusAuthorView.swift

```diff
@@ -152,25 +152,30 @@ extension StatusAuthorView {
     }
 
     public func setupAuthorMenu(menuContext: AuthorMenuContext) -> (UIMenu, [UIAccessibilityCustomAction]) {
-        var actions: [MastodonMenu.Action] = []
-
-        actions = [
-            .muteUser(.init(
-                name: menuContext.name,
-                isMuting: menuContext.isMuting
-            )),
-            .blockUser(.init(
-                name: menuContext.name,
-                isBlocking: menuContext.isBlocking
-            )),
-            .reportUser(
-                .init(name: menuContext.name)
-            ),
+        var actions = [MastodonMenu.Action]()
+
+        if !menuContext.isMyself {
+            actions.append(contentsOf: [
+                .muteUser(.init(
+                    name: menuContext.name,
+                    isMuting: menuContext.isMuting
+                )),
+                .blockUser(.init(
+                    name: menuContext.name,
+                    isBlocking: menuContext.isBlocking
+                )),
+                .reportUser(
+                    .init(name: menuContext.name)
+                )
+            ])
+        }
+        
+        actions.append(contentsOf: [
             .bookmarkStatus(
                 .init(isBookmarking: menuContext.isBookmarking)
             ),
             .shareStatus
-        ]
+        ])
```

Comment:
Was your placement of this append to make sure the order of the `MastodonMenu.Action` array matches the procedural ordering?